src/views/workflow-summary-tab/workflow-summary-tab-json-view/workflow-summary-tab-json-view.types.ts

```diff
@@ -4,4 +4,9 @@ export type Props = {
   inputJson: PrettyJsonValue;
   resultJson: PrettyJsonValue;
   isWorkflowRunning: boolean;
+  isArchived: boolean;
+  domain: string;
```

Comment:
There's a reusable type for WorkflowPageParams somewhere that you can use here. You can check the Workflow Diagnostics UI code for reference